src/work-allocation-2/containers/task-list/task-list.component.spec.ts

```diff
@@ -429,7 +429,7 @@ describe('TaskListComponent', () => {
     let paginationSummary: HTMLElement;
 
     beforeEach(() => {
-      paginationSummary = fixture.debugElement.nativeElement.querySelector('#search-result-summary__text');
+      paginationSummary = fixture.debugElement.nativeElement.querySelector('span.text-16');
```

Comment:
Adding another span with a text-16 class would break this test. If we need to target a specific element then the element should have an id.

Reply:
I don't get why we need to add another selector just to accommodate testing. Surely if someone adds another span they should update the tests. And you shouldn't use ids target tests maybe custom attributes would suffice.